packages/commons/src/context/context.ts

```diff
@@ -15,15 +14,14 @@ import { AuthData } from "../auth/authData.js";
 import { genericLogger, Logger, logger } from "../logging/index.js";
 import { parseCorrelationIdHeader } from "../auth/headers.js";
 
-export const AppContext = z.object({
-  serviceName: z.string(),
-  authData: AuthData,
-  correlationId: CorrelationId,
-  requestTimestamp: z.number(),
-});
-export type AppContext = z.infer<typeof AppContext>;
+export type AppContext<A extends AuthData = AuthData> = {
+  serviceName: string;
+  authData: A;
+  correlationId: CorrelationId;
+  requestTimestamp: number;
+};
 
-export const zodiosCtx = zodiosContext(z.object({ ctx: AppContext }));
+export const zodiosCtx = zodiosContext();
```

Comment:
Unneeded, we never used this zodios feature, we just set the context in the request manually and we behave knowing it's there in further middlewares/handlers